extensions/wikia/PhalanxII/classes/PhalanxBlockTestPager.class.php

```diff
@@ -10,19 +10,19 @@ public function __construct( $blockType ) {
 		$this->blockType = $blockType;
 	}
 
-	public function setRows(Array $rows) {
+	public function setRows( Array $rows ) {
 		$this->rows = $rows;
 	}
 
 	public function getHeader() {
-		return Html::element('h2', array(), sprintf('%s (%d)', $this->blockType, count($this->rows)));
+		return Html::element( 'h2', array(), sprintf( '%s (%d)', $this->blockType, count( $this->rows ) ) );
 	}
 
 	public function getBody() {
 		$ret = '<ul>';
 
-		foreach($this->rows as $row) {
-			$ret .= $this->formatRow($row);
+		foreach ( $this->rows as $row ) {
+			$ret .= $this->formatRow( $row );
 		}
 
 		$ret .= '</ul>';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$ret". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)